src/docs/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,33 @@
+= Semantic Data Structuring Working Group - Documentation 
```

Comment:
This does not seem right, why are there two `index.adoc`s with similar content?

Reply:
You are right, that there was an issue because I forgot to delete the index.adoc under modules/ROOT which is also the reason for the strange text in the other index.adoc.